src/main/java/de/unijena/cheminf/mortar/model/util/CollectionUtil.java

```diff
@@ -34,14 +34,15 @@
  */
 public final class CollectionUtil {
     //<editor-fold desc="public static methods" defaultstate="collapsed">
+    //TODO: check parameters, change params to enum values (and bools?), throw exceptions!
     /**
      * Sorts given list by property and sort type.
      *
      * @param aList List
      * @param aProperty String
      * @param aSortType String
      */
-    public static void sortGivenFragmentListByPropertyAndSortType(List<MoleculeDataModel> aList, String aProperty, String aSortType) {
+    public static void sortGivenFragmentListByPropertyAndSortType(List<? extends MoleculeDataModel> aList, String aProperty, String aSortType) {
         Collections.sort(aList, (m1, m2) -> {
```

Comment:
Thanks, I will refactor this methods later

Reply:
That would be nice but I could also do it. Let's discuss this later after this PR has been merged.